### PR TITLE
fix talk command, add some helpful tips

### PR DESCRIPTION
**Pull request by Jigoku**
Changes proposed in this request:
- One of the tips doesn't display the keyboard emblems due to misnamed binds.
- Add some new helpful tips

### CODE REVIEW

File: config/tips.cfg [hunk 1]
```diff
@@ -31,6 +31,12 @@ tipreset = [
     tipadd "You can chat with the community and developers in ^fs^fcDiscord^fS at ^fs^fcwww.redeclipse.net/discord^fS."
     tipadd "Share your own tips with the developers in ^fs^fcDiscord^fS at ^fs^fcwww.redeclipse.net/discord^fS."
     tipadd "Tips are ^fs^fccool^fS, you should ^fs^fyread them more often^fS!"
+    tipadd "When a teammate is ^fs^foon fire^fS, you can extinguish the flames by using the ^fs^fyflamer^fS secondary weapon."
```
Comment (qreeves) [inline config/tips.cfg]: Specify the key to press: `{=secondary}`

File: config/tips.cfg [hunk 2]
```diff
@@ -31,6 +31,12 @@ tipreset = [
...
+    tipadd "The ^fs^fyzoom level^fS of a scoped weapon can be adjusted by ^fs^fyscrolling^fS."
```
Comment (qreeves) [inline config/tips.cfg]: Probably want `{=universaldelta 1}` and `{=universaldelta -1}`

File: config/tips.cfg [hunk 3]
```diff
@@ -31,6 +31,12 @@ tipreset = [
...
+    tipadd "Press ^fs^fw^f{=special}^fS whilst running to quickly ^fs^fyvault^fS across ledges or small obstacles."
+    tipadd "Press ^fs^fw^f{=crouch}^fS and ^fs^fw^f{=jump}^fS together whilst in mid-air to perform an ^fs^fyimpulse pound^fS."
+    tipadd "In ^fs^fybomber-ball^fS mode, you can ^fs^fypass^fS the bomb to teammates by pressing ^fs^fw^f{=affinity}^fS."
```
Comment (qreeves) [inline config/tips.cfg]: Pressing the button quickly will lob the bomb without passing, you have to *hold* the button and have the aiming reticle display for it to successfully pass.

File: config/tips.cfg [hunk 4]
```diff
@@ -31,6 +31,12 @@ tipreset = [
...
+    tipadd "Taking your own ^fs^fyflag^fS in ^fs^fycapture^fS mode will make it harder for your opponents to steal."
```
Comment (qreeves) [inline config/tips.cfg]: Probably want to mention `{=affinity}` here as well.

Comment (Jigoku) [reply config/tips.cfg]: should be done now

**Pull request merged**